frontend/src/components/ScheduleCreation.vue

```diff
@@ -847,9 +851,9 @@ watch(
     :open="refreshSlugModalOpen"
     :title="t('label.refreshLink')"
     :message="t('text.refreshLinkNotice')"
-    :confirm-label="t('label.refresh')"
+    :confirm-label="t('label.save')"
```

Comment:
At this place, I found "save" to be more convenient, since we continue to save data here instead of just refreshing the link.